plugins/traction_innkeeper/pyproject.toml

```diff
@@ -9,11 +9,12 @@ packages = [{include = "traction_innkeeper"}]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-aries-cloudagent = { version = "0.11.0" }
+aries-cloudagent = { version = "0.12.rc3" }
```

Comment:
We should probably hold on this to align with 0.12.0 release before merging to Traction

Reply:
Interesting :+1:

Reply:
Traction main branch (base images and innkeeper plugin dependencies) are updated to 0.12.0 now